compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -409,6 +409,28 @@ impl StatementKind<'_> {
     }
 }
 
+#[derive(
+    Clone,
+    TyEncodable,
+    TyDecodable,
+    Debug,
+    PartialEq,
+    Hash,
+    HashStable,
+    TypeFoldable,
+    TypeVisitable
+)]
```

Comment:
Nit: please split the derives in 2 lines.

Reply:
Do you mean like this?
```
#[derive(
    Clone, TyEncodable, TyDecodable, Debug, PartialEq,
    Hash, HashStable, TypeFoldable, TypeVisitable
)]
```

Reply:
I mean:
```
#[derive(Clone, TyEncodable, TyDecodable, Debug, PartialEq)]
#[derive(Hash, HashStable, TypeFoldable, TypeVisitable)]
```
To avoid rustfmt using a dozen lines.

Reply:
Done